### ISSUE TITLE
make runtest error 134 - infile.is_open() 

### ISSUE THREAD

**Issue by rbrth** [BVLC/caffe#3576]
When I make runtest, I get the following error which I cannot seem to pinpoint its origin. 

[ RUN      ] SoftmaxWithLossLayerTest/2.TestGradientWeights
F0120 13:54:50.808677 20147 softmax_loss_layer.cpp:29] Check failed: infile.is_open() 
*** Check failure stack trace: ***
    @     0x2b6e6cb55daa  (unknown)
    @     0x2b6e6cb55ce4  (unknown)
    @     0x2b6e6cb556e6  (unknown)
    @     0x2b6e6cb58687  (unknown)
    @           0x83bd4d  caffe::SoftmaxWithLossLayer<>::LayerSetUp()
    @           0x434e9f  caffe::Layer<>::SetUp()
    @           0x436f1f  caffe::GradientChecker<>::CheckGradientExhaustive()
    @           0x5be0ef  caffe::SoftmaxWithLossLayerTest_TestGradientWeights_Test<>::TestBody()
    @           0x723f23  testing::internal::HandleExceptionsInMethodIfSupported<>()
    @           0x71aa67  testing::Test::Run()
    @           0x71ab0e  testing::TestInfo::Run()
    @           0x71ac15  testing::TestCase::Run()
    @           0x71df58  testing::internal::UnitTestImpl::RunAllTests()
    @           0x71e1e7  testing::UnitTest::Run()
    @           0x423c2a  main
    @     0x2b6e6fbaeec5  (unknown)
    @           0x42ddd7  (unknown)
    @              (nil)  (unknown)
Aborted (core dumped)

**Comment by chenhuahui1993** [comment]
I have the same problem with you. Did you solve this problem?

**Comment by rbrth** [comment]
@chenhuahui1993 , I just ignored the make runtest and assumed/hoped the error would not pop up during runtime of caffe. Hence not really solved..

**Comment by shelhamer** [comment]
From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:

> When reporting a bug, it's most helpful to provide the following information, where applicable:
> 
> * What steps reproduce the bug?
> * Can you reproduce the bug using the latest [master](https://github.com/BVLC/caffe/tree/master), compiled with the `DEBUG` make option?
> * What hardware and operating system/distribution are you running?
> * If the bug is a crash, provide the backtrace (usually printed by Caffe; always obtainable with `gdb`).

**Comment by rbrth** [comment]
@shelhamer You are right, I should've specified more information. Thank you for the hints.
